Pipfile

```diff
@@ -4,12 +4,13 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-django = "*"
-django-extensions = "*"
-psycopg2 = "*"
+django = "==3.2.10"
+django-extensions = "==3.1.4"
+djangorestframework = "==3.11.2"
+psycopg2 = "==2.8.3"
 
 [dev-packages]
-werkzeug = "*"
+werkzeug = "==0.16.0"
```

Comment:
These are locked to mirror our trident deps. Using wildcarding here wasn't a great idea, since it would pull in Django 4 despite the project being built for django 3.2.